### ISSUE TITLE
Is it dead?? Not working on marshmallow

### ISSUE THREAD

**Issue by ghost** [cmrn/squire#12]
I used it a lot now it doesn't work!

**Comment by cmrn** [comment]
Yeah sorry this is no longer maintained, and I've lost the Android signing key to update the version published in the app store. Anyone is free to take the source code, continue development & republish it, within the terms of the license.
